File: 프로그래머스/unrated/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    let result = n;
    const CONTROL_ARRAY = control.split('');
    for(let key of CONTROL_ARRAY) {
        switch(key) {
            case "w" :
                result++;
                break;
            case "s" :
                result--;
                break;
            case "d" :
                result += 10;
                break;
            case "a" :
                result -= 10;
                break;
        }
    }
    return result;
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.py
def solution(n, control):
    answer = n
    
    for com in control:
        if com == "w":
            answer += 1
        elif com == "s":
            answer -= 1
        elif com == "d":
            answer += 10
        elif com == "a":
            answer -= 10
    
    return answer